### PR TITLE
14 articles auto block

### LINKED ISSUE #14: articles [auto] block

### PR DESCRIPTION
**Pull request by dkuntze**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #14

Test URLs:
- Before: https://main--basf-retailfirst--hlxsites.hlx.page/expected-ag-trends-2022
- After: https://14-articles-auto-block--basf-retailfirst--hlxsites.hlx.page/expected-ag-trends-2022

### CODE REVIEW

File: blocks/header/header.css [hunk 1]
```diff
@@ -41,15 +43,18 @@ header nav div:first-child {
 header nav > div:nth-child(2) {
   height: 170px;
 }
+
 header nav > div:nth-child(2) p {
-  margin: 0 0 -6px 0;
```
Comment (ravkiran) [inline blocks/header/header.css:45]: do you want to remove the `margin-left` here?

File: styles/styles.css [hunk 2]
```diff
@@ -238,6 +242,59 @@ main img {
...
+
+body.article div.article-hero h1 {
+    font-size: 60px;
+}
+
+body.article div.article-body {
+    font-size: var(--body-font-size-xs);
+}
+
+body.article div.article-body li strong {
```
Comment (ravkiran) [inline styles/styles.css:270]: you may need to add the same thing for selector `article div.article-body p strong`

**Review by ravkiran — APPROVED** [review]
LGTM

File: blocks/header/header.css (hunk 1 above)
Comment (dkuntze) [reply blocks/header/header.css:45]: lets fix this in another PR

File: styles/styles.css (hunk 2 above)
Comment (dkuntze) [reply styles/styles.css:270]: we'll open another issue/pr if needed.

**Pull request merged**